### PR TITLE
Closes#4386 added Installation instructions for Docker Extension

### LINKED ISSUE #4386: [Meshery] Installation instructions for Docker Extension
### Current Behavior
Docker Extension installation instructions are missing on https://layer5.io/cloud-native-management/meshery/getting-started

### Desired Behavior
The same instructions listed on [https://meshery.io#getting-started](https://meshery.io/#getting-started) should be available on this page.

### Screenshots / Mockups
Screenshot 2023-06-16 at 8 45 41 AM

### Implementation
Copy/paste from meshery.io

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ranjanmangla1**
**Description**
Added Installation instructions for Docker Extension on the "https://layer5.io/cloud-native-management/meshery/getting-started" page.

This PR fixes #4386 
changed in : src/sections/Meshery/Meshery platforms/index.js
Added installation instructions for Docker Extension given exactly on [https://meshery.io#getting-started](https://meshery.io/#getting-started)

**Notes for Reviewers**
Earlier the installation steps for Docker looked like: 
image

Now it looks like as follows: 
image

Responsivity of the page is maintained and you can check by clicking on other platform icons like Kubernetes.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit be6b56c66718b4ac34d2171394cf89ba1bc514ec at: https://64943e4083840b208a66603e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 96b58a8d79cf65070b8c0d2f9a9b2720c4111cd3 at: https://649445e102ef9d2a17be9958--layer5.netlify.app

**Comment by Savio629** [comment]
@ranjanmangla1 Please mention fixes #4386 below the description!
No need to add it in the title of the pr.

**Comment by ranjanmangla1** [comment]
> @ranjanmangla1 Please mention fixes #4386 below the description! No need to add it in the title of the pr.

@Savio629 added

File: src/sections/Meshery/Meshery-platforms/index.js [hunk 1]
```diff
@@ -42,6 +42,10 @@ const supported_platforms = [
         <Code codeString={dedent`curl -L https://meshery.io/install | PLATFORM=docker bash -`
         }
         />
+        <h2 style={{ marginTop: "20px" }}>Docker Extension</h2>
+        <Code codeString={dedent`docker extension install meshery/docker-extension-meshery:stable-latest`
```
Comment (vishalvivekm) [inline src/sections/Meshery/Meshery-platforms/index.js:46]: @ranjanmangla1 We've a dedicated page for Meshery docker extension: https://layer5.io/docker-extension-meshery, can you include a link to it here. 
for example: "know more about Meshery docker extension" and then the link

Comment (ranjanmangla1) [reply src/sections/Meshery/Meshery-platforms/index.js:46]: So, I should remove the installation command instead and add the link instead?

Comment (thisiskaransgit) [reply src/sections/Meshery/Meshery-platforms/index.js:46]: @ranjanmangla1 NO, add the text redirecting to the page, between the heading and command.

Comment (leecalcote) [reply src/sections/Meshery/Meshery-platforms/index.js:46]: Great suggestion to tie in a link to the dedicated Extensions page https://layer5.io/docker-extension-meshery. Let's forego its inclusion here and seek out other pages to interlink. 👍

**Comment by leecalcote** [comment]
Thanks for this PR, @ranjanmangla1 👏

**Pull request merged**